Documentation/SystemModules.md

```diff
@@ -39,7 +39,7 @@ Create a directory called `CJPEG` parallel to the `example` directory and create
 Edit the `module.modulemap` so it consists of the following:
 
     module CJPEG [system] {
-        header "/usr/include/jpeglib.h"
+        header "/usr/local/include/jpeglib.h"
```

Comment:
I think it's better to keep this at `/usr/include/jpeglib.h` and mention that on OS X it may differ. Or mention that it may be found inside `/usr/include` on Linux, since it's very unlikely to be found at `/usr/local` anywhere but OS X.

These changes make this document OS X specific.

For example, for me:

```shell
$ ls -l /usr/include/jpeglib.h 
-rw-r--r-- 1 root root 49594 Dec  5 21:50 /usr/include/jpeglib.h
$ ls -l /usr/local/include/jpeglib.h 
ls: cannot access /usr/local/include/jpeglib.h: No such file or directory
```